### PR TITLE
Remove typings.d.ts #1604

### PR DESCRIPTION
**Pull request by miguel4ngelc**

### CODE REVIEW

**Comment by filipesilva** [comment]
@hansl can you review if this is what you wanted?
@miguel4ngelc tests seem to be broken, can you fix?

File: addon/ng2/blueprints/ng2/files/__path__/test.ts [hunk 1]
```diff
@@ -14,6 +14,14 @@ import 'zone.js/dist/sync-test';
 // Unfortunately there's no typing for the `__karma__` variable. Just declare it as any.
 declare var __karma__: any;
 
+// Typings reference file, see links for more information
```
Comment (hansl) [inline addon/ng2/blueprints/ng2/files/__path__/test.ts]: These comments are useless. Please provide a comment that explains why we need those declaration here.

**Comment by hansl** [comment]
The tests are broken by this change. Is there anything else that needs those typings? There shouldn't.

**Comment by filipesilva** [comment]
Related issue: https://github.com/angular/angular-cli/issues/1604

**Comment by hansl** [comment]
Thanks, @filipesilva, I forgot about that one. I'm not entirely sure how to approach this now, but having `typings.d.ts` is still kinda wrong. Let's talk about it internally and come up with the right solution.

**Comment by filipesilva** [comment]
The tests are passing, but I actually see a lot of errors on the log itself: https://travis-ci.org/angular/angular-cli/jobs/152746267#L2320. This is the error we were expecting when converting template imports via https://github.com/TheLarkInn/angular2-template-loader.

More importantly though, @TheLarkInn it seems that `ng build` with errors are not giving the correct exit code.

**Comment by Brocco** [comment]
Outdated a new PR is being submitted to update this file, but it is needed to reference System and to be available to specify other typings files for global libraries.

**Pull request closed without merging**